content/en/agent/guide/community-integrations-installation-with-docker-agent.md

```diff
@@ -20,55 +20,27 @@ Community developed integrations for the Datadog Agent are stored in the [Integr
 
 To install the `<INTEGRATION_NAME>` check on your host:
 
-1. Install the [developer toolkit][1].
-2. Clone the integrations-extras repository:
+1. [Download and launch the Datadog Agent][1].
+2. Run the following command to install the integrations with the Agent:
 
     ```
-    git clone https://github.com/DataDog/integrations-extras.git.
+    datadog-agent integration install -t <INTEGRATION_NAME>==<INTEGRATION_VERSION>
```

Comment:
Shouldn't there be `-r` here too?

Reply:
The reason why I didn’t put `-r` here is because it wasn’t present in the old instructions that this one is superseding:
```
datadog-agent integration install -w <PATH_OF_INTEGRATION_NAME_PACKAGE>/<ARTIFACT_NAME>.whl
```

So, I suppose that those instructions are supposed to be executed with the unprivileged `dd-user` user.

Am I right ?

Reply:
^ Thoughts @ofek @FlorianVeaux ?

Reply:
Makes sense, though have you tried it?

Reply:
Yes, the `dd-user` is enough, except in containerzaied environments where root is required. But in that case, the command becomes `agent` instead of `datadog-agent`.
We already document both methods so the change looks good.